binder/src/test/java/io/grpc/binder/RobolectricBinderSecurityTest.java

```diff
@@ -148,19 +122,13 @@ private static MethodDescriptor<Empty, Empty> getMethodDescriptor() {
         .build();
   }
 
-  private static class SomeService extends LifecycleService {
-
-    private final IBinderReceiver binderReceiver = new IBinderReceiver();
-    private final ArrayBlockingQueue<SettableFuture<Status>> statusesToSet =
-        new ArrayBlockingQueue<>(128);
-    private Server server;
-    private final ScheduledExecutorService scheduledExecutorService =
-        new MainThreadScheduledExecutorService();
-
-    @Override
-    public void onCreate() {
-      super.onCreate();
+  private final IBinderReceiver binderReceiver = new IBinderReceiver();
```

Comment:
Nit: these fields feel out of place interlaved between instance methods. We should probably hoist them up within the class.

go/java-practices/class-structure#general

Reply:
Tried to leave them in their same relative position to minimize the diff but you are right. I moved them to the top.